API/Controllers/OfferController.cs

```diff
@@ -27,5 +27,15 @@ public async Task<IActionResult> Create([FromBody] OfferCreateDTO offer)
             var userId = _userService.GetCurrentUserNameIdentifier(User);
             return Ok(await _offerService.CreateOfferAsync(offer, userId));
         }
+
+        [HttpGet("offer-info")]
+        public async Task<IActionResult> GetOfferInfo(int offerId)
+        {
+            var userId = _userService.GetCurrentUserNameIdentifier(User);
+
+            OfferInfoDTO offerDTO = await _offerService.GetConcreteOfferAsync(offerId, userId);
```

Comment:
You do not have to specify the type of data being returned. You can just write «var»

Reply:
Resolved